.clang-format

```diff
@@ -0,0 +1,34 @@
+---
+BasedOnStyle: Google
+---
+Language: Cpp
+ColumnLimit: 100
+IndentWidth: 8
+ContinuationIndentWidth: 8
+ConstructorInitializerIndentWidth: 8
+TabWidth: 8
+UseTab: ForIndentation
```

Comment:
No, the decision eventually fell back to tabs. I have tried to make the settings indent-only, so that alignment should in theory never come up (for example, function arguments are indented, not aligned).

Reply:
Actually, I was lying, I hadn't. But now I have! (`UseTab` setting)